src/sugar3/activity/widgets.py

```diff
@@ -396,3 +395,4 @@ def __init__(self):
         self.paste = PasteButton()
         self.insert(self.paste, -1)
         self.paste.show()
+
```

Comment:
Please do not stylistic include changes to this file, unless you have other changes.